doc/BBCode.md

```diff
@@ -1,208 +1,611 @@
-Example:
-
-<pre>[abstract]Totally interesting! A must-see! Please click the link![/abstract]
-I want to tell you a really boring story that you really never wanted 
-to hear.</pre>
-
-Twitter would display the text "Totally interesting! A must-see! Please click the link!". 
-On Friendica you would only see the text after "I want to tell you a really ..."
+<table class="bbcodes">
+<tr>
+  <th>BBCode</th>
+  <th>Result</th>
+</tr>
+<tr>
+  <td>[video]url[/video]</td>
+  <td>Where *url* can be an url to youtube, vimeo, soundcloud, or other sites wich supports oembed or opengraph specifications.</td>
+</tr>
+<tr>
+  <td>[video]Video file url[/video]
+[audio]Audio file url[/audio]</td>
+  <td>Full URL to an ogg/ogv/oga/ogm/webm/mp4/mp3 file. An HTML5 player will be used to show it.</td>
+</tr>
+<tr>
+  <td>[youtube]Youtube URL[/youtube]</td>
+  <td>Youtube video OEmbed display. May not embed an actual player.</td>
+</tr>
+<tr>
+  <td>[youtube]Youtube video ID[/youtube]</td>
+  <td>Youtube player iframe embed.</td>
+</tr>
+<tr>
+  <td>[vimeo]Vimeo URL[/vimeo]</td>
+  <td>Vimeo video OEmbed display. May not embed an actual player.</td>
+</tr>
+<tr>
+  <td>[vimeo]Vimeo video ID[/vimeo]</td>
+  <td>Vimeo player iframe embed.</td>
+</tr>
+<tr>
+  <td>[iframe]URL[/iframe]</td>
+  <td>General embed, iframe size is limited by the theme size for video players.</td>
+</tr>
+<tr>
+  <td>[url]*url*[/url]</td>
+  <td>If *url* supports oembed or opengraph specifications the embedded object will be shown (eg, documents from scribd).
+Page title with a link to *url* will be shown.</td>
+</tr>
+</table>
+
+## Map
+
+This require "openstreetmap" addon version 1.3 or newer. If the addon isn't activated,
+the raw coordinates are shown instead.
+
+<table class="bbcodes">
+<tr>
+  <th>BBCode</th>
+  <th>Result</th>
+</tr>
+<tr>
+  <td>[map]address[/map]</td>
+  <td>Embeds a map centered on this address.</td>
+</tr>
+<tr>
+  <td>[map=lat,long]</td>
+  <td>Embeds a map centered on those coordinates.</td>
+</tr>
+<tr>
+  <td>[map]</td>
+  <td>Embeds a map centered on the post's location.</td>
+</tr>
+</table>
+
+## Abstract for longer posts
+
+If you want to spread your post to several third party networks you can have the problem that these networks have a length limitation like on Twitter.
+
+Friendica is using a semi intelligent mechanism to generate a fitting abstract.
+But it can be interesting to define a custom abstract that will only be displayed on the external network.
+This is done with the [abstract]-element.
+<table class="bbcodes">
+<tr>
+  <th>BBCode</th>
+  <th>Result</th>
+</tr>
+<tr>
+  <td>[abstract]Totally interesting! A must-see! Please click the link![/abstract]<br>
+I want to tell you a really boring story that you really never wanted to hear.</td>
+  <td>Twitter would display the text "Totally interesting! A must-see! Please click the link!".
+On Friendica you would only see the text after "I want to tell you a really ..."</td>
+</tr>
```

Comment:
For my taste, there should be a paragraph around what will be seen on Twitter and what will be shown on Friendica in the result column.